ros/src/computing/perception/detection/lidar_tracker/packages/lidar_imm_ukf_pda_track/nodes/imm_ukf_pda/imm_ukf_pda.cpp

```diff
@@ -763,7 +763,7 @@ void ImmUkfPda::tracker(const autoware_msgs::DetectedObjectArray& input,
       continue;
     }
     // prevent ukf not to explode
-    if (targets_[i].p_merge_.determinant() > prevent_explosion_thres_ ||
+    if (targets_[i].p #ifndef_merge_.determinant() > prevent_explosion_thres_ ||
```

Comment:
@cirpue49 Is this modification intended?

Reply:
That is not intentional., Thanks for the help.